docs/PROVISIONING_AWS.adoc

```diff
@@ -0,0 +1,142 @@
+* Create a link:https://console.aws.amazon.com/route53/home?#hosted-zones:[Hosted Zone] in your AWS Account.
+
+=== Example Policy Assignment
+
+AWS Policy assignment can be pain. While perhaps not a minimalist  solution, here is an example set of policies that will allow provisioning to function.
+
+* link:https://console.aws.amazon.com/iam/home?#policies/arn:aws:iam::aws:policy/AmazonEC2FullAccess[EC2FullAccess]
+* link:https://console.aws.amazon.com/iam/home?#policies/arn:aws:iam::aws:policy/AmazonVPCFullAccess[VPCFullAccess]
+* link:https://console.aws.amazon.com/iam/home?#policies/arn:aws:iam::aws:policy/AmazonRoute53FullAccess[Route53FullAccess]
+
+Additionally, the following custom policy must be created for provisioning of CloudFormation stacks.
+
+[source,json]
+----
+{
+  "Version": "2012-10-17",
+  "Statement": [
+    {
+      "Sid": "Stmt1535729505000",
+      "Effect": "Allow",
+      "Action": [
+        "cloudformation:*"
+      ],
+      "Resource": [
+        "arn:aws:cloudformation:us-east-2:[Insert ARN]:stack/*"
+      ]
+    }
+  ]
+}
+----
+
+NOTE: Make sure the region matches your target region above and that you insert an ARN that matches your account.
+
+Cool! Now you're ready to provision OpenShift clusters on AWS
+
+== Provision an OpenShift Cluster
+
+As an example, we'll provision the `sample.aws.example.com` cluster defined in the `~/src/casl-ansible/inventory` directory.
+
+NOTE: It is recommended that you use a different inventory similar to the ones found in the `~src/casl-ansible/inventory` directory and keep it elsewhere. This allows you to update/remove/change your casl-ansble source directory without losing your inventory. Also note that it may take some effort to get the inventory just right, hence it is very beneficial to keep it around for future use without having to redo everything.
+
+The following is just an example on how the `sample.aws.example.com` inventory can be used:
+
+1. Select an link:https://docs.aws.amazon.com/AmazonRDS/latest/UserGuide/Concepts.RegionsAndAvailabilityZones.html[AWS Region] you would like to provision your cluster in.
+  * Modify the `regions` entry (line 13) in the inventory `ec2.ini` with your region
+  * Modify the `cloud_infrastructure.region` variable in `group_vars/all.yml` with your region.
+2. Select an _environment id_. This is a shortname for your cluster. In the example cluster domain `sample.aws.example.com`, your _environment id_ would be `sample`.
+  * Modify `instance_filters` entry (line 14) in the inventory `ec2.ini` file to match your chosen _environment id_
+  * Set the `env_id` variable in `group_vars/all.yml` to match.
+3. Edit `group_vars/all.yml` to match your AWS environment. See comments in the file for more detailed information on how to fill these in.
+4. Edit `group_vars/OSEv3.yml` for your AWS specific configuration. See comments in the file for more detailed information on how to fill these in.
```

Comment:
Probably should reword "... for your AWS specific configuration." to something like "... for OpenShift Container Platform related configuration."